### PR TITLE
DOC: Fix typo disutils -> distutils in numpy.distutils migration page

### PR DESCRIPTION
**Pull request by lesteve**
Everything is in the title I think :wink:

### CODE REVIEW

File: doc/source/reference/distutils_status_migration.rst [hunk 1]
```diff
@@ -111,7 +111,7 @@ For more details, see the
 
 .. _numpy-setuptools-interaction:
 
-Interaction of ``numpy.disutils`` with ``setuptools``
+Interaction of ``numpy.distutils`` with ``setuptools``
 -----------------------------------------------------
```
Comment (melissawm) [inline doc/source/reference/distutils_status_migration.rst:115]:
```suggestion
Interaction of ``numpy.distutils`` with ``setuptools``
------------------------------------------------------
```

**Review by melissawm — COMMENTED** [review]
Hi @lesteve - thanks for the PR! The docs build is failing because of an underline character that is now missing from the section title. Otherwise, looks good 😄

File: doc/source/reference/distutils_status_migration.rst (hunk 1 above)
Comment (lesteve) [reply doc/source/reference/distutils_status_migration.rst]: Indeed, thanks!

**Comment by seberg** [comment]
Thanks @lesteve and Melissa, lets put this in!

**Pull request merged**